torch/onnx/symbolic_helper.py

```diff
@@ -247,19 +249,65 @@ def _interpolate_size_to_scales(g, input, output_size, dim):
     output_size = _maybe_get_const(output_size, 'is')
     if _is_value(output_size):
         offset = 2
-        offsets = g.op("Constant", value_t=torch.ones(offset))
+        offsets = g.op("Constant", value_t=torch.ones(offset, dtype=torch.float32))
         dividend = g.op("Cast", output_size, to_i=cast_pytorch_to_onnx["Float"])
-        divisor = _slice_helper(g, g.op("Shape", input), axes=[0], ends=[dim], starts=[offset])
+        divisor = _slice_helper(g, g.op("Shape", input), axes=[0], ends=[9223372036854775807], starts=[offset])
```

Comment:
Can we use some constant like double.max to replace this number? it's so error prone.